gcloud/bigquery/test_table.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=too-many-lines
```

Comment:
Still need to address:

1. Let's just raise our allowed number of lines for test files (we have a separate test config). What is the other test file that has too many lines?
1. Maybe this is a sign that `table.py` is doing too much and we should break up the band.